src/js/relevant.js

```diff
@@ -203,8 +203,16 @@ export default {
              */
             let context = p.path;
             if (
-                getChild(node, `.or-repeat-info[data-name="${p.path}"]`) &&
-                !getChild(node, `.or-repeat[name="${p.path}"]`)
+                (getChild(node, `.or-repeat-info[data-name="${p.path}"]`) &&
+                    !getChild(node, `.or-repeat[name="${p.path}"]`)) ||
+                // Repeat instance removed for model node with no visible form control
+                (insideRepeat && repeatParent == null && options.removed) ||
+                // All repeat instances removed for model node with no visible form control, i.e. during load with `jr:count="0"`
+                (repeatPath != null &&
```

Comment:
Is `repeatPath != null` the same as `insideRepeat`?

Reply:
`insideRepeat` should be used here (it is equivalent to `repeatPath != null && p.path.startsWith(`${repeatPath}`);`).